kview/src/main/java/dev/ikm/komet/kview/mvvm/view/navigation/PatternNavEntryController.java

```diff
@@ -145,6 +161,14 @@ private void initialize() {
         setupListView();
     }
 
+    void initializeTooltip() {
+        Platform.runLater(() -> {
+            if (!patternName.isTextTruncated()) {
+                Tooltip.uninstall(patternName, displayNameTooltip);
+            }
```

Comment:
try doing the applyCSS and layout here before calling isTruncated, if isTruncated is not returnint the correect values. 
We shouldn't need a platform.runLater here.